roles/gluster_hci/README.md

```diff
@@ -22,7 +22,8 @@ Role Variables
 | gluster_features_hci_port || 24007 | The port to be set for the remote  |
 | gluster_features_fqdn_check |true/false|true|Check if the hosts have valid FQDN. Set the variable to false to skip the FQDN check.|
 | gluster_features_min_disk ||15360|The minimum disk size that /var/log directory must have for the role to proceed. By default the role checks for 15G. Should be in megabytes|
-| gluster_features_force_diskcheck |true/false|true| Force the role to enforce the minimum disk size requirement for /var/log. If set to false, disk size check is ignored.|
+| gluster_features_force_diskcheck |true/false|true|Force the role to enforce the minimum disk size requirement for /var/log. If set to false, disk size check is ignored.|
```

Comment:
variable name is misleading..suggests that the check is about disks

Reply:
Ack! Will rename it to gluster_features_foce_varsizecheck